clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/InternalLocalBootstrappingVeniceChangelogConsumer.java

```diff
@@ -470,15 +466,20 @@ public CompletableFuture<Void> seekWithBootStrap(Set<Integer> partitions) {
         }
       }
 
-      // Seek to the current position so we can catch up from there to target
-      seekToCheckpoint(
-          bootstrapStateMap.values().stream().map(state -> state.currentPubSubPosition).collect(Collectors.toSet()));
+      // Seek to the current position, so we can catch up from there to target
+      try {
+        seekToCheckpoint(
+            bootstrapStateMap.values().stream().map(state -> state.currentPubSubPosition).collect(Collectors.toSet()))
+                .get();
+      } catch (Exception e) {
+        throw new VeniceException("Caught exception when seeking to bootstrap", e);
+      }
 
       // Poll until we've caught up completely for all subscribed partitions.
       while (bootstrapStateMap.entrySet()
           .stream()
           .anyMatch(s -> s.getValue().bootstrapState.equals(PollState.CATCHING_UP))) {
-        pollAndCatchup(5000L, ChangeCaptureView.CHANGE_CAPTURE_TOPIC_SUFFIX);
+        pollAndCatchup(5000L, "");
```

Comment:
Let's add a todo here similar to the PR I opened, we might down the road need to tweak this, but for now it's fine since we're extending AfterImageConsumer specifically (so theres an implication that this only works on that type of topic).